### ISSUE TITLE
small usability issue: Result 1 to Query Status

### ISSUE THREAD

**Issue by wellsoliver** [jakob/Postico#118]
I often check the query status view to see notifications, etc, and find it a little frustrating to click the Result 1 drop-down and select it. Would it be possible to add a button that allows a quick toggle?

(i know, tiny, minor issue)

**Comment by qwesda** [comment]
+1 this is especially true when developing functions

**Comment by trapias** [comment]
I'd better view a tabview here, where tabs are "query status", "results", and then maybe others? 

pgAdmin has an output pane like this:

<img width="339" alt="schermata 2015-11-27 alle 11 46 44" src="https://cloud.githubusercontent.com/assets/2881610/11439573/a6571862-94fc-11e5-8f6d-6566a2d875fd.png">

**Comment by jakob** [comment]
Tabs are indeed a good idea here, considering that most of the time there will be few results. If you execute a query with lots of results that will fail, though. But I can always display some kind of overflow tab with a popup in that case...

**Comment by trapias** [comment]
why would it fail? the table with records should just be put within a tab, like in pgadmin.

Do you mean it will fail on UI?

**Comment by jakob** [comment]
I was talking about the case when you have multiple resultsets. If you execute multiple queries at once, you will get multiple tables. Using the popup menu you can switch between a lot of them, but using tabs you could only switch between 5 or so, since you won't fit more than 5 tabs or so on the screen.

This is probably not a common case, but I try to make sure Postico works whatever you do. If you execute a query that returns 100 results, Postico should handle it. As I said, using tabs, I could probably use an overflow tab to handle this case.

**Comment by trapias** [comment]
Ah got it, right!

Worst case: you could keep a single tab for results, and append all result sets as separate tables within it, would this work? Not optimal to read results, yet.

Else an "overflow" tab (I imagine something like a "more tabs" tab, with buttons to move left and right to see more tabs) would be perfect, and I believe usable in most of cases. 

Nothing against current switch indeed, but feel a tabview would allow me to move from "results" to "messages" more quickly. just this. What about a keyboard shortcut to select "query status" or "result n", like first 5 results, keeping current switch?

**Comment by jakob** [comment]
You are right about tabs being quicker. I think the tab bar is the way to go; but I'm working on some other stuff right now, so this will have to wait a little. Also, good point about the keyboard shortcuts, those would definitely be helpful as well.

**Comment by trapias** [comment]
Waiting for next release, thanks @jakob :smile:

**Comment by qwesda** [comment]
I'm currently fighting the interface a lot with this one.

I have some helper functions to manipulate the db and they raise some notices, which I'm interressted in. The return type is `void`, but since 1 row with 1 column is returned postico switches to the result viewer.

May be you could check for this case and disable the switching if the result set is 1 row with 1 column and `pg_typeof("column_1") = 'void'` (not sure how/if this is exposed in libpq though).

**Comment by wildfluss** [comment]
+1
